extensions/amp-recaptcha-input/0.1/amp-recaptcha-input.js

```diff
@@ -94,6 +112,22 @@ export class AmpRecaptchaInput extends AMP.BaseElement {
     }
     return true;
   }
+
+  /**
+   * Function to return the recaptcha token.
+   * Will be an override of AMP.AsyncInput
+   */
+  getValue() {
+    if (this.sitekey_ && this.action_) {
```

Comment:
Yes, it will only be called after `buildCallback`.

I looked, and there is no built in in Base element for guarding this functionality. So yes, lets do that in our next PR 😄